lib/util.js

```diff
@@ -253,8 +250,7 @@ function arrayToHash(array) {
   var hash = Object.create(null);
 
   for (var i = 0; i < array.length; i++) {
-    var val = array[i];
-    hash[val] = true;
+    hash[array[i]] = true;
```

Comment:
Likely.. still, it's an unnecessary assign. exceedingly minor but since I was in here anyway...